R/backend-mssql.R

```diff
@@ -637,6 +631,8 @@ mssql_bit_int_bit <- function(f) {
 
 #' @export
 `db_sql_render.Microsoft SQL Server` <- function(con, sql, ..., cte = FALSE, use_star = TRUE) {
+  check_unsupported_arg(cte, allowed = FALSE)
```

Comment:
I think you shouldn't check those here because we're still calling the next method (which might support them)?